src/general/OptionsMenu.cs

```diff
@@ -769,6 +775,54 @@ private Viewport.MSAA MSAAIndexToResolution(int index)
         }
     }
 
+    private int MaxFPSValueToIndex(int value)
+    {
+        switch (value)
+        {
+            case 30:
+                return 0;
+            case 60:
+                return 1;
+            case 90:
+                return 2;
+            case 120:
+                return 3;
+            case 144:
+                return 4;
+            case 240:
+                return 5;
+            case 0:
```

Comment:
400, and 1000 should be the highest given options. as the max FPS is partly meant to avoid graphics cards melting if they render thousands of FPS.